frontend/tests/components/Checkbox.test.tsx

```diff
@@ -7,7 +7,7 @@ import "regenerator-runtime/runtime";
 import { config } from "react-transition-group";
 
 // Internal imports
-import { Checkbox } from "components/components";
+import Checkbox from "tw-components/Checkbox";
```

Comment:
```suggestion
import { Checkbox } from "tw-components/Checkbox";
```
The Checkbox component uses named export, `export { Checkbox }`, so it should be imported with `{}`